api/proto/teleport/integration/v1/awsoidc_service.proto

```diff
@@ -257,3 +263,42 @@ message ListEC2Response {
   // If non-empty, it can be used to request the next page.
   string next_token = 2;
 }
+
+// ListEKSClustersRequest is a request for a paginated list of AWS EKS Clusters.
+message ListEKSClustersRequest {
+  // Integration is the AWS OIDC Integration name.
+  // Required.
+  string integration = 1;
+  // Region is the AWS Region
+  // Required.
+  string region = 2;
+  // NextToken is the token to be used to fetch the next page.
+  // If empty, the first page is fetched.
+  string next_token = 3;
+}
+
+// EKSCluster represents an Amazon EKS Cluster.
+message EKSCluster {
+  // Name is the name of AWS EKS cluster.
+  string name = 1;
+  // Region is an AWS region.
+  string region = 2;
+  // Arn is an AWS ARN identification of the EKS cluster.
+  string arn = 3;
+  // Labels are labels of a EKS cluster.
+  map<string, string> labels = 4;
```

Comment:
by labels you mean AWS tags, is that correct? 
If yes, tags support null values

Reply:
I would expect the converter to have an empty string.
However, this ends up being the same as is exposed in the SDK:
https://github.com/aws/aws-sdk-go-v2/blob/7888f0e10eb7ef5db0b67a8a27bb8eeee984257b/service/eks/types/types.go#L356-L359

Also, this is mostly exposing the internal API as a gRPC service, and this section was mostly a copy-paste from https://github.com/gravitational/teleport/blob/943bef63d92bbd06c0ca2ad018f2e25d5a80a4a2/lib/integrations/awsoidc/eks_list_clusters.go#L66-L67